sphinx_selective_exclude/eager_only.py

```diff
@@ -31,9 +31,13 @@ def run(self, *args):
         env = self.state.document.settings.env
         env.app.builder.tags.add('TRUE')
         #print(repr(self.arguments[0]))
-        if not env.app.builder.tags.eval_condition(self.arguments[0]):
-            return []
-
+        try:
+            if not env.app.builder.tags.eval_condition(self.arguments[0]):
+                return []
+        except Exception as err:
+            logger = logging.getLogger(__name__)
```

Comment:
this should be a global variable of the file, or a class-member, or property or whatever. Having it here in the except only, will force new logs to copy this line around...

Reply:
> this should be a global variable

+1. And please initialize it as `logger = sphinx.util.logging.getLogger(__name__)` to avoid confusion of what it is.

Reply:
I have done this now.

Sorry I missed this comment.